### DISCUSSION TITLE
How to convert local files

### DISCUSSION THREAD

**Discussion by Jimmy2024Aus** [aclap-dev/video-downloadhelper#906] [Help and Q&A]
### Your question

I have VDH version 9.0.2.12a1 installed, and when I click the "Convert Local Files" button, I don't get the choices of file types to convert. It lists only Mp4, MKV, and WEBM. I have an ".rm" file on my computer I would like to convert to MP4, but I don't know how. I'm missing something, but I don't know what. Any help would be appreciated. I DO have a current license for Firefox and am using the Firefox browser. Thanks

### Addon & CoApp version and details

I have VDH version 9.0.2.12a1 installed, and when I click the "Convert Local Files" button, I don't get the choices of file types to convert. It lists only Mp4, MKV, and WEBM. I have an ".rm" file on my computer I would like to convert to MP4, but I don't know how. I'm missing something, but I don't know what. Any help would be appreciated. I DO have a current license for Firefox and am using the Firefox browser. Thanks Version 9.0.2.12a1 (beta), target mozilla, locale en-US.

Companion App installed (v2.0.19) Recheck

C:\Program Files\DownloadHelper CoApp\vdhcoapp.exe

License verified (jimmy2001tx@hotmail.com)

**Comment by paulrouget** [comment]
What if you click on MP4, and then select you `.rm` file?

**Comment by Jimmy2024Aus** [comment]
I'll try that. Thank you, sir.

On Wed, Jun 19, 2024 at 10:26 PM Paul ***@***.***> wrote:

> What if you click on MP4, and then select you .rm file?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/aclap-dev/video-downloadhelper/discussions/906#discussioncomment-9823866>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BIWR42WVYZSTPI6YV53MRE3ZIJDUTAVCNFSM6AAAAABJS2NG6KVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TQMRTHA3DM>
> .
> You are receiving this because you authored the thread.Message ID:
> <aclap-dev/video-downloadhelper/repo-discussions/906/comments/9823866@
> github.com>
>


-- 
*J.R.*




*"You'll never be criticized by someone who is doing more than you. You'll
always be criticized by someone doing less. Remember that."<Denzel
Washington>*
